psi4/driver/procrouting/empirical_dispersion.py

```diff
@@ -121,17 +122,17 @@ def __init__(self, name_hint=None, level_hint=None, param_tweaks=None, **kwargs)
         from .dft import dashcoeff_supplement
         self.dashcoeff_supplement = dashcoeff_supplement
 
-        resolved = intf_dftd3.from_arrays(
+        resolved = qcng.programs.dftd3.from_arrays(
             name_hint=name_hint,
             level_hint=level_hint,
             param_tweaks=param_tweaks,
             dashcoeff_supplement=self.dashcoeff_supplement)
         self.fctldash = resolved['fctldash']
         self.dashlevel = resolved['dashlevel']
         self.dashparams = resolved['dashparams']
-        self.description = intf_dftd3.dashcoeff[self.dashlevel]['description']
-        self.ordered_params = intf_dftd3.dashcoeff[self.dashlevel]['default'].keys()
-        self.dashlevel_citation = intf_dftd3.dashcoeff[self.dashlevel]['citation']
+        self.description = qcng.programs.dftd3.dashcoeff[self.dashlevel]['description']
```

Comment:
Would recommend changing this to:
```py
dftd3 = qcng.get_program('dftd3')
self.desc = dftd3.dashcoef...
```

Reply:
which is the concern -- full word "description" or that `qcng.programs.dftd3` isn't a forever address?

Reply:
Sorry no real preference on `desc`, but moving everything the `dftd3` object would be good. I think its fine for now, but pushing things away from free floating functions is why we built the classes.